L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCTriggerCells.cc

```diff
@@ -148,6 +166,16 @@ fill(const edm::Event& e, const edm::EventSetup& es)
   std::unordered_map<uint32_t, double> simhits_bh;  
   if(fill_simenergy_) simhits(e, simhits_ee, simhits_fh, simhits_bh);
 
+  edm::Handle<CaloToCellsMap> caloparticles_map_h;
+  std::map<uint32_t, unsigned> cell_to_genparticle;
```

Comment:
Better to use an `unordered_map` if the order is not important

Reply:
fixed.